docs/content/how-to/azure/create-azure-cluster_on_aks.md

```diff
@@ -126,22 +126,120 @@ nodePoolManagement=$(az ad sp create-for-rbac --name "${NODEPOOL_MGMT}" --create
 ```
 
 ### 6. Save Service Principal and Key Vault Details
```

Comment:
Can you add some text/blurb here about what's being accomplished? Including some details about MIv2 and MIv3 differences in setup.